python/ml4ir/base/config/parse_args.py

```diff
@@ -151,6 +151,20 @@ def define_args(self):
             help="Run ID for the current training. Autogenerated if not specified.",
         )
 
+        self.add_argument(
+            "--run_group",
+            type=str,
+            default="general",
+            help="Run group defining a meta grouping to filter different model training runs for best model selection as a post step.",
+        )
+
+        self.add_argument(
+            "--run_notes",
+            type=str,
+            default="",
+            help="Notes for the current training run. Use this argument to add short description of the model training run that helps in identifying the run later.",
```

Comment:
+1, could we also say: where do the note end up? And how can the user get back to it?

Reply:
It is just going to be added to the experiment tracking data. Currently it is in the weights and biases summary file. But we might choose to write a JSON of our own later.

Reply:
So in the help we need to say what we currently do, and update it when we change it :)

Reply:
@lastmansleeping Ya, it would be helpful to add that this note goes in the weights and biases summary file